fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rule/transformation/materialization/TableScanDesc.java

```diff
@@ -28,12 +28,16 @@ public class TableScanDesc {
     private final LogicalScanOperator scanOperator;
     // join type of LogicalJoinOperator above scan operator
     private final JoinOperator parentJoinType;
+    private final boolean isLeft;
```

Comment:
what about full outer join

Reply:
full outer join is not supported in view delta join rewrite.
I fix it in inner join case.